substrate/primitives/arithmetic/src/fixed_point.rs

```diff
@@ -175,7 +175,7 @@ pub trait FixedPointNumber:
 		d: D,
 	) -> Option<Self> {
 		if d == D::zero() {
-			return None
+			return None;
```

Comment:
Probably your IDE doing this; but you can improve review speed by omitting unrelated changes.